CONTRIBUTING.md

```diff
@@ -38,6 +38,12 @@ git clone https://github.com/microsoft/hummingbird.git .
 pip install -e .
 ```
 
+We also provide a simple [Dockerfile](https://github.com/microsoft/hummingbird/blob/master/Dockerfile) that you can customize to your preferred development environment. This Dockerfile will create a Jupyter notebook environment and also install development tools such as `flake8` and `coverage`.
```

Comment:
Do you think this should be here, or on the main readme under install instructions?